.github/workflows/ci.yaml

```diff
@@ -74,12 +85,26 @@ jobs:
           pip freeze
           deactivate
 
+      # Run nox session
       - name: Run nox
         env:
           NOX_SESSION: ${{ matrix.nox-session }}
-        run: nox -R -e "$NOX_SESSION"
+        run: |
+          if [ "$NOX_SESSION" = "pytest_min" ]; then
+            nox -db uv -R -e "$NOX_SESSION" -- --ff -o cache_dir=.pytest_cache
+          else
+            nox -db uv -R -e "$NOX_SESSION"
+          fi
         timeout-minutes: 10
 
+      # Save pytest cache only for pytest_min session
+      - name: Save pytest cache
+        if: always() && matrix.nox-session == 'pytest_min'
```

Comment:
No, it's actually working, I tested this PR extensively :)

![image](https://github.com/user-attachments/assets/f09315b7-dd6a-4e1c-a082-e6939d3c1f36)

Reply:
ok, I see it works.
I read the [docs](https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/using-conditions-to-control-job-execution) too and that confirms it too, though there are exceptions for cases that this won't work.
```
When you use expressions in an if conditional, you can, optionally, omit the ${{ }} expression syntax because GitHub Actions automatically evaluates the if conditional as an expression. However, this exception does not apply everywhere.

You must always use the ${{ }} expression syntax or escape with '', "", or () when the expression starts with !, since ! is reserved notation in YAML format. For example:
```

I've also seen in line 66 it is written `if: ${{ matrix.nox-session == 'pytest_min' }}` and ideally we should be consistent on how these statements are written.
IMO it feels safer to always use `${{ }}`, and I agree it doesn't look good